### DISCUSSION TITLE
v3.2.85

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#13020] [General]
## What's Changed
* Fix: page title formatting by @KeizerDev in https://github.com/filamentphp/filament/pull/13017
* Change css invisibility strategy in Wizard steps by @sebapastore in https://github.com/filamentphp/filament/pull/12985
* Fixes `Method Filament\Tables\Actions\Action::isAutofocused does not exist`.

## New Contributors
* @KeizerDev made their first contribution in https://github.com/filamentphp/filament/pull/13017

**Full Changelog**: https://github.com/filamentphp/filament/compare/v3.2.84...v3.2.85

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v3.2.85'>v3.2.85</a>.</em>

**Comment by fatihdurmaz** [comment]
Value in Rich Editor not being reset.
![Ekran görüntüsü 2024-06-02 215505](https://github.com/filamentphp/filament/assets/71706970/59634b6d-6955-40c8-a931-03375f68089a)
